frontend/src/components/MapFiltersMenu/MapFiltersMenu.tsx

```diff
@@ -91,7 +99,8 @@ const MapFiltersMenu = ({
                     <CacheTypeFilter onChange={modifyCacheTypes} cacheTypes={mapFilters.cacheTypes} eventKey="2"/>
                     <CacheSizeFilter onChange={modifyCacheSize} size={mapFilters.size} eventKey="3"/>       
                     <NameContainsFilter onChange={modifyNameContains} nameContains={mapFilters.nameContains} eventKey="4"/>
-                    <DifficultyFilter onChange={modifyDifficulty} difficulty={mapFilters.difficulty} eventKey="6"/> 
+                    <DifficultyFilter onChange={modifyDifficulty} difficulty={mapFilters.difficulty} eventKey="6"/>
+                    <AttributesFilter onChange={modifyAttributes} attributes={mapFilters.attributes} eventKey="7"/>
```

Comment:
We need to configure eventKeys and sequence. I was following Figma. So when you merge from main, we should put them in order, and use eventKeys accordingly. 1-4 are like this in Main, 5 is Description contains, 6 Difficulty, 7 Terrain. So attributes should be 8th, and I'll put Published/Hidden as 9, the last one here.

Reply:
ok